ExeterTeam25/group-software-project
# Sso v2

To Test:
- pip install msal
- pip install PyYAML
- pip install python-dateutil
- add the oauth_settings.yml file to the project file structure
- make sure no account has your uni email tied to it, if so delete and use another email for that account
- go to login
- sign in with Microsoft 
- enter your uni email
- you should be logged in as the new user
- complete a challenge
- logout
- log back in with Microsoft login
- check that the correct user has been logged in

Details:
- added functionality to login with a university email
- updated readme
